### PR TITLE
FS-1528-Send-Round-Name-To-Notification

### PR DESCRIPTION
**Pull request by gio-karunakaran**
Posting round_name instead of round_id to Notification service

### CODE REVIEW

File: external_services/models/notification.py [hunk 1]
```diff
@@ -28,18 +28,18 @@ def send(template_type: str, to_email: str, content: dict):
...
-                "Sorry, the notification could not be sent for endpoint:"
-                f" '{url}', params: '{params}'."
-            )
+        # if response:
+        current_app.logger.info(
+            "application sent to notification service with response:"
+            f" '{response}'."
         )
+        return response
+        # raise NotificationError(
```
Comment (donshellywork) [inline external_services/models/notification.py:37]: commented code shouldn't be here

**Review by SentientHousePlant — APPROVED** [review]
LGTM but remove dead code before merging.

File: external_services/models/notification.py (hunk 1 above)
Comment (gio-karunakaran) [reply external_services/models/notification.py]: done!

**Review by AbuNavsa — APPROVED** [review]
Brill!

**Pull request merged**